collectors/lib/enrich_events_metric_reporter.py

```diff
@@ -0,0 +1,38 @@
+import sys
+
+from samza_metric_reporter import SamzaMetricReporter
+
+class EnrichEventsMetricReporter(SamzaMetricReporter):
```

Comment:
This is nearly identical to `SessionizationMetricReporter`. Only difference is `m = ` let's just parameterize `SessionizationMetricReporter`